account_check_printing_report_base/tests/test_check_printing_report.py

```diff
@@ -138,15 +132,11 @@ def test_check_printing_with_layout(self):
         company and journal."""
 
         self.company.check_layout_id = self.check_report
-        acc_payable = self._create_account(
-            "account payable test", "ACPRB1", self.acc_payable, True
-        )
-        vendor_bill = self._create_vendor_bill(acc_payable)
         acc_expense = self._create_account(
             "account expense test", "ACPRB2", self.acc_expense, False
         )
-        self._create_invoice_line(acc_expense, vendor_bill)
-        vendor_bill.action_invoice_open()
+        vendor_bill = self._create_vendor_bill(acc_expense)
+        vendor_bill.state = "posted"
```

Comment:
I think you should use action_post method instead
```suggestion
        vendor_bill.action_post()
```